### PR TITLE
refactor code

### PR DESCRIPTION
**Pull request by themousepotato**
# Fixes # 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Fix the two issues mentioned in https://codeclimate.com/github/sendgrid/sendgrid-python/sendgrid/helpers/mail/mail.py#issue_59f2b5d6a7918a000100005e given in #461
- 

If you have questions, please send an email to [SendGrid](mailto:dx@sendgrid.com), or file a GitHub Issue in this repository.

### CODE REVIEW

**Comment by thinkingserious** [comment]
Hello @themousepotato,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @themousepotato,<br /><br/>Thanks again for the PR!<br /><br />                                         We noticed that you have earned 4 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold.<br /><br />                                         Also, please be sure that you have officially registered <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid DX

**Comment by themousepotato** [comment]
Hey @thinkingserious ! I sent some other PRs too. But, the travis build failed due to error: no environment variables found. If this cannot get resolved, I might miss the swag by just one point. :(

**Comment by thinkingserious** [comment]
Hello @themousepotato,<br /><br/>Thanks again for the PR!<br /><br />                                         You have earned 4 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold or to simply continue the celebration.<br /><br />                                         Also, <strong>please be sure you have officially registered with us <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a> by November 1, 2018</strong> to qualify.<br /><br />                                         If you have any questions you can email us at <a href="mailto:dx+hacktoberfest2018@sendgrid.com">dx+hacktoberfest2018@sendgrid.com</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid

File: sendgrid/helpers/mail/mail.py [hunk 1]
```diff
@@ -65,6 +66,13 @@ def __str__(self):
         """
         return str(self.get())
 
+    def __update_params(self, given_args, mail, arg_str):
```
Comment (42B) [inline sendgrid/helpers/mail/mail.py]: Using double-underscores is not usually what you want to do, wouldn't a single preceding underscore achieve the same goal here?

**Comment by childish-sambino** [comment]
Fixed by https://github.com/sendgrid/sendgrid-python/commit/6c232045fc7cee179aeae16d75bdaaaad6cd09f4

**Pull request closed without merging**